.eleventy.js

```diff
@@ -42,40 +42,78 @@ module.exports = function (eleventyConfig) {
+      'app/posts/register-trainee-teachers/register-trainee-teachers.md',
+      'app/posts/support-for-apply/support-for-apply.md',
+      'app/posts/support-for-publish/support-for-publish.md'
+    ])
+  })
+
+  // Collections of posts for each service
   eleventyConfig.addCollection('apply-for-teacher-training', collection => {
-    return collection.getFilteredByTag('apply-for-teacher-training').filter(item => {
-      return !item.data.tags.includes('user-need')
-    })
+    return collection.getFilteredByGlob('app/posts/apply-for-teacher-training/*.md')
+  })
+
+  eleventyConfig.addCollection('find-teacher-training', collection => {
+    return collection.getFilteredByGlob('app/posts/find-teacher-training/*.md')
   })
 
   eleventyConfig.addCollection('manage-teacher-training-applications', collection => {
-    return collection.getFilteredByTag('manage-teacher-training-applications').filter(item => {
-      return !item.data.tags.includes('user-need')
-    })
+    return collection.getFilteredByGlob('app/posts/manage-teacher-training-applications/*.md')
   })
 
   eleventyConfig.addCollection('publish-teacher-training-courses', collection => {
-    return collection.getFilteredByTag('publish-teacher-training-courses').filter(item => {
-      return !item.data.tags.includes('user-need')
-    })
+    return collection.getFilteredByGlob('app/posts/publish-teacher-training-courses/*.md')
   })
 
   eleventyConfig.addCollection('register-trainee-teachers', collection => {
-    return collection.getFilteredByTag('register-trainee-teachers').filter(item => {
-      return !item.data.tags.includes('user-need')
-    })
+    return collection.getFilteredByGlob('app/posts/register-trainee-teachers/*.md')
   })
 
   eleventyConfig.addCollection('support-for-apply', collection => {
-    return collection.getFilteredByTag('support-for-apply').filter(item => {
-      return !item.data.tags.includes('user-need')
-    })
+    return collection.getFilteredByGlob('app/posts/support-for-apply/*.md')
   })
 
   eleventyConfig.addCollection('support-for-publish', collection => {
-    return collection.getFilteredByTag('support-for-publish').filter(item => {
-      return !item.data.tags.includes('user-need')
-    })
+    return collection.getFilteredByGlob('app/posts/support-for-publish/*.md')
+  })
+
+  // A collection of reference pages
+  eleventyConfig.addCollection('reference', collectionApi => {
+    return collectionApi.getFilteredByGlob(['app/glossary.md', 'app/mission-patches.md', 'app/service-map.md'])
+  })
+
+  // A collection of reference pages
+  eleventyConfig.addCollection('allTags', collectionApi => {
+    const posts = collectionApi.getAll()
+
+    // Set up empty list of tags
+    let tags = []
+
+    for (post of posts) {
+      if ('tags' in post.data) {
+        // Add any new tags from the post to the array
+        for (tag of post.data.tags) {
+          if (!tags.includes(tag)) {
+            tags.push(tag)
+          }
```

Comment:
I think this might be case sensitive?

Reply:
Yeah it is. I think I’m ok with that for now. Lowercasing everything would mean "UCAS" becomes "ucas", and doing a case-insensitive check when building the array would mean that whichever version appeared first would be how it appears in the tags list.  This way would at least let us spot when we’ve been consistent with the casing...?

Reply:
It'll mean we have case sensitive urls, and would try to generate pages at `/tags/Foo` and `/tags/foo`.

Reply:
ok so I’ve added some extra code which will throw a helpful error if this happens so that we can catch it.